modules/OFStateManager/module/src/gentable_handlers.c

```diff
@@ -1235,6 +1273,7 @@ indigo_core_gentable_entry_resume(
         /* incorrect resume call */
         AIM_LOG_ERROR("%s: should have valid obj (no_async=%d)",
                       __FUNCTION__, no_async);
+        aim_free(op_ctx);
         return;
```

Comment:
wouldn't something be seriously wrong if the object were NULL in the op_ctx?  maybe assert here instead?